### PR TITLE
Ensure cleanup of temp directories

### PR DESCRIPTION
**Pull request by atobiszei**
JIRA:CVS-73268

### CODE REVIEW

File: client/python/ovmsclient/lib/scripts/build_tfs_api.sh [hunk 1]
```diff
@@ -18,6 +18,15 @@
 
 set -e
 
+finally() {
```
Comment (mzegla) [inline client/python/ovmsclient/lib/scripts/build_tfs_api.sh:21]: Shouldn't it be:
`function finally() {`
?
Also I would change it's name to something like "remove_tmp_dirs" as "finally" looks a bit like some keyword or special value.

Comment (jasiu86) [reply client/python/ovmsclient/lib/scripts/build_tfs_api.sh:21]: God point, make function name more descriptive

Comment (jasiu86) [reply client/python/ovmsclient/lib/scripts/build_tfs_api.sh:21]: cleanup ?
final_cleanup ?

Comment (atobiszei) [reply client/python/ovmsclient/lib/scripts/build_tfs_api.sh]: function keyword is not required. Changed the function name to be more descriptive.

**Review by mzegla — APPROVED** [review]

**Review by jasiu86 — APPROVED** [review]

**Pull request merged**